lib/rake/build.rake

```diff
@@ -16,14 +30,10 @@ namespace :build do
       # Workaround for https://github.com/karma-runner/karma-phantomjs-launcher/issues/120
       RakeUtils.npm_rebuild 'phantomjs-prebuilt'
 
-      if rack_env?(:staging)
-        ChatClient.log 'Updating <b>apps</b> i18n strings...'
-        RakeUtils.system './sync-apps.sh'
-      end
-
       ChatClient.log 'Building <b>apps</b>...'
       npm_target = (rack_env?(:development) || ENV['CI']) ? 'build' : 'build:dist'
       RakeUtils.system "npm run #{npm_target}"
+      File.write(apps_dir('commit_hash'), RakeUtils.git_folder_hash(apps_dir))
```

Comment:
It shouldn't interfere, since the packager unconditionally writes `commit_hash` just before packaging. Any existing `commit_hash` in the `/apps` source-folder won't be included in the commit since this file is added to `.gitignore`.